cgi-bin/DW/Routing.pm

```diff
@@ -556,6 +576,53 @@ sub register_api_endpoints {
     }
 }
 
+=head2 C<< $class->register_api_rest_endpoint( $string, $sub, %opts ) >>
+
+=over
+
+=item string
+
+=item sub - sub
+
+=item opts (see register_regex)
+
+=back
+
+=cut
+
+sub register_api_rest_endpoint {
+    my ( $class, $string, $sub, $controller_class, %opts ) = @_;
+
+    # this is dev only for now -akp
+    return unless $LJ::IS_DEV_SERVER;
```

Comment:
do we want this still?

Reply:
And this one, do we still want this? Or, maybe maybe this some `$LJ::ENABLE_REST_API`?